src/accountingservice/README.md

```diff
@@ -18,3 +17,23 @@ From the root directory, run:
 ```sh
 docker compose build accountingservice
 ```
+
+## Regenerate protos
+
+> **Note**
+> [`protoc`](https://grpc.io/docs/protoc-installation/) is required.
+
+To regenerate gRPC code run:
+
+```sh
+go generate
+```
```

Comment:
Do we want to add a job to the "Checks" GitHub workflow that would make sure that the committed generated code is up-to-date? Do we want to do this in this same PR?

Reply:
Let’s do a separate PR if you wouldn’t mind at least creating an issue for it. 

I’ll go ahead and merge this